### ISSUE TITLE
Fullscreen, controls and resolution

### ISSUE THREAD

**Issue by idcero** [Afterster/videojs-vlc#3]
When i go fullscreen i cant see the controls in chrome. (works in firefox)
Video keeps the height and width when i go into fullscreen, same on both browsers.

using this css and videojs:
//vjs.zencdn.net/4.12/video-js.css
//vjs.zencdn.net/4.12/video.js

and your example

**Comment by farwarx** [comment]
Hi,
In fullscreen, the video don't expend :/
It was working before the poster fix.

**Comment by jkaberg** [comment]
Example: 
http://i.imgur.com/UEdIKLZ.png
Fullscreen: http://i.imgur.com/I9rPV8z.png

@Afterster

**Comment by hryuk** [comment]
+1

Video doesn't extend in fullscreen or even if you change the video player size, it keeps the size it has at the first render.

**Comment by ghost** [comment]
anyone got fix for this yet?

**Comment by Afterster** [comment]
I didn't get much time for this yet sorry, I will finish another project first and then take a look again on this issue. Always a time problem :)

**Comment by ghost** [comment]
Understood it's awesome plugin. My users like it better then the actual vlc plugin.

**Comment by ghost** [comment]
Another issue it has is it won't autoplay.

**Comment by Afterster** [comment]
After several testing, this is not related to poster changes but to _windowless_ option set to true (and it has to be to display video.js control bar).
I applied a fix suggested on https://trac.videolan.org/vlc/ticket/12790, this will add some hang when going/leaving fullscreen but it should fix the issue. @idcero @farwarx @hzeroo @imdummyhelpme, could you confirm it is working correctly on your side too?

**Comment by ghost** [comment]
I will let you know when i get home thanks for bug fix i will give you an update . Were you able to fix autoplay issue?thanks for your help just saw libs you did add an autoplay i will try it and will give you an update.

**Comment by Afterster** [comment]
Yes autoplay issue should be fixed too. Let me know about your results when you get some time to test it.
